sea-orm-macros/Cargo.toml

```diff
@@ -18,8 +18,8 @@ path = "src/lib.rs"
 proc-macro = true
 
 [dependencies]
-bae = { version = "0.1", default-features = false }
-syn = { version = "1", default-features = false, features = ["parsing", "proc-macro", "derive", "printing"] }
+bae2 = { version = "1", default-features = false }
```

Comment:
Since `bae` crate is no longer maintained and it is required that bae is upgrade to syn v2 too, so I made a fork of bae and published to crates.io.

Reply:
Is there any alternative to `bae`?

Reply:
@billy1624
I think we can adopt darling after https://github.com/TedDriggs/darling/pull/226 got merged. Also it is not so difficult to parse attributes without bae or darling, thanks to `parse_nested_meta` API. In fact, major of sea_orm attributes are parsed manually without using bae.

Reply:
I was thinking we use Darling since it has nicer errors when you get a name wrong